src/applications/disability-benefits/all-claims/config/submitForm.js

```diff
@@ -1,12 +1,15 @@
 import { transformForSubmit } from 'platform/forms-system/src/js/helpers';
 import recordEvent from 'platform/monitoring/record-event';
 import * as Sentry from '@sentry/browser';
+import localStorage from 'platform/utilities/storage/localStorage';
```

Comment:
I don't think that we should have to import this just so that the tests will pass. I'm curious to see if others have any opinions on it.

Reply:
yeah, I agree

Reply:
I think it's fine. Don't really have strong opinions on it. A lot of other tests are using this `localStorage` already.

Reply:
If the fallback `localStorage` is no longer necessary, it can be removed in a separate PR along with all the other instances where it's used.